Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+edition = "2021"
+name = "scylladb-javascript-driver"
+version = "0.0.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+# Default enable napi4 feature, see https://nodejs.org/api/n-api.html#node-api-version-matrix
+napi = { version = "2.12.2", default-features = false, features = ["napi4", "async"] }
+napi-derive = "2.12.2"
+scylla = { git = "https://github.com/scylladb/scylla-rust-driver.git", rev = "v0.14.0", features = [
+    "ssl",
+] }
+tokio = { version = "1.34", features = ["full"] }
+futures = "0.3"
+
+
+[build-dependencies]
+napi-build = "2.0.1"
+
+[profile.release]
+lto = true
+strip = "symbols"
+
+[profile.dev]
+panic = "abort"
```

Comment:
That's interesting. Is this the preferred way to handle panics when using napi? I would expect that napi-rs enables some conversion between Rust panics and JS exceptions, enabling unified backtraces. Backtraces can be very useful to debug Rust driver's issues when used from JS, and with this setting we lose them completely.

Reply:
We didn't (yet) look at panicking of rust driver. We will probably address this later.